pwndbg/commands/telescope.py

```diff
@@ -39,24 +40,27 @@ def telescope(address=None, count=telescope_lines, to_string=False):
     ($sp by default)
     """
     address = int(address if address else pwndbg.regs.sp) & pwndbg.arch.ptrmask
-    count   = int(count) & pwndbg.arch.ptrmask
+    count   = max(int(count), 0) & pwndbg.arch.ptrmask
```

Comment:
If `count` is zero, do we do anything at all?  Might be better to bail with an error than use `max(..., 0)` since it's a no-op if `count == 0`.